### PR TITLE
fix(completions): add wallbash sub completions

### PR DESCRIPTION
**Pull request by rubiin**
Adds the wallbash options value for toggle

### CODE REVIEW

**Comment by rubiin** [comment]
Btw what are the `mode` string names ? I couldnt find them

File: shell-completions/Hyde.usage
Comment (kRHYME7) [inline shell-completions/Hyde.usage:1]: Sorry for the confusion 

wallbsh toggle only does toggle

walbash mode is a selector from 1 2 3 4 . if no strings for mode then will do the GUI thing 

image

**Comment by rubiin** [comment]
https://github.com/kRHYME7/Hyde-cli/blob/aee0e0282dff29bfa869fe464e0d4820b00d701e/Hyde#L295C1-L298C74

I think this needs update as well. If toggle doesnt need argument. 
Also `Hyde wallbash mode` without any argument does not open gui but gives this

image

**Comment by kRHYME7** [comment]
It works on my end 

you got this https://github.com/kRHYME7/Hyde-cli/blob/aee0e0282dff29bfa869fe464e0d4820b00d701e/Hyde#L301C1-L301C76 ?

**Comment by rubiin** [comment]
I will remove the options from toggle then if it does not need any options

**Comment by kRHYME7** [comment]
yes please,

**Comment by rubiin** [comment]
Updated can be merged. The zsh completions doesnt seem to be working from the package but when I source the .zsh file , it does work . Have to look into the folders that zsh sources the completions from.
Since for zsh , we have ohmyzsh installed, we can use its folder instead

**Comment by rubiin** [comment]
https://github.com/ohmyzsh/ohmyzsh/discussions/11342

**Comment by rubiin** [comment]
lets chat on discord about the issue with zsh

**Pull request merged**